backend/pkg/routing/routing.go

```diff
@@ -106,3 +106,16 @@ func NewGinRouter() *gin.Engine {
 
 	return router
 }
+
+// New Gin router without any middlewares
+func NewMinimalGinRouter() *gin.Engine {
+	SwitchToReleaseMode()
+
+	router := gin.New()
+
+	router.HandleMethodNotAllowed = true
+	router.NoMethod(handleNoMethod)
+	router.NoRoute(handleNoRoute)
+
+	return router
+}
```

Comment:
All endpoints (public or internal) should have accesslogs and requestid.

Reply:
This PR is removing accesslogs entirely from internal endpoints which is not what we want. I don't think it's that important to suppress the from successful health checks.